Autonomous_Parallel_Parking/Autonomous_Parallel_Parking.c

```diff
@@ -15,14 +15,84 @@
  ****************************************************************************************************************/
 
 #include "cpu.h"
-#include "board_led.h"
+#include "pin_setup.h"
 
-#include "foo.h"
+int count = 0;
 
 int main()
 {
   cpu_init();
-  board_led_init();
-
+  GPIOInterruptSetup();
   return 0;
 }
+
+/**
+* @brief IRQ for PA0 (user button) - to test if PA0 interrupt is functioning.  Turns on blue LED every 5 times the 
+* button is pressed.
+*/
+void EXTI0_IRQHandler(void){
```

Comment:
Hey do we need to keep this? This is just to test if your code was running right? Will we use this going forward?